source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRView.mm

```diff
@@ -307,10 +311,15 @@ - (void)processTextConcurrently:(std::shared_ptr<BaseCardElement> const &)textEl
             NSString* parsedString = nil;
             std::shared_ptr<MarkDownParser> markDownParser = nullptr;
 
-            if(CardElementType::TextBlock == elementTypeForBlock){
-                std::shared_ptr<TextBlock> textBlockElement = std::dynamic_pointer_cast<TextBlock>(textElementForBlock);
-                markDownParser = std::make_shared<MarkDownParser>([ACOHostConfig getLocalizedDate:textBlockElement]);
-            } else {
+            if(CardElementType::TextBlock == elementTypeForBlock)
+            {
```

Comment:
>  { [](start = 10, length = 3)

`{` should be at the end of the preceding line. in general, modifications to existing code should follow the existing coding style.